build.gradle.kts

```diff
@@ -823,16 +846,16 @@ tasks {
 
   register("copyCatalogLibAndConfigs", Copy::class) {
     dependsOn(
+      ":catalogs:catalog-hadoop:copyLibAndConfig",
       ":catalogs:catalog-hive:copyLibAndConfig",
-      ":catalogs:catalog-lakehouse-iceberg:copyLibAndConfig",
-      ":catalogs:catalog-lakehouse-paimon:copyLibAndConfig",
-      "catalogs:catalog-lakehouse-hudi:copyLibAndConfig",
       ":catalogs:catalog-jdbc-doris:copyLibAndConfig",
       ":catalogs:catalog-jdbc-mysql:copyLibAndConfig",
       ":catalogs:catalog-jdbc-oceanbase:copyLibAndConfig",
       ":catalogs:catalog-jdbc-postgresql:copyLibAndConfig",
-      ":catalogs:catalog-hadoop:copyLibAndConfig",
       ":catalogs:catalog-kafka:copyLibAndConfig",
+      ":catalogs:catalog-lakehouse-hudi:copyLibAndConfig",
```

Comment:
To reviewers: 'catalogs:catalog-lakehouse-hudi:copyLibAndConfig' was missing the leading ':'.

Reply:
It's fine, these two ways are both workable for Gradle.